extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/values.yaml

```diff
@@ -52,6 +52,6 @@ eventVolume:
   mountOptions:
     - allow-delete
   size: 1Gi
-  volumePathOnNode: /tmp
+  volumePathOnNode: /tmp/spark-events/
```

Comment:
Does this not need to match `spark.eventLog.dir: "/opt/spark/spark-events"`?

Reply:
It does not.  This references the path on the node itself, rather than the pod's FS.